rudder-core/src/main/scala/com/normation/rudder/services/policies/write/Cf3PromisesFileWriterService.scala

```diff
@@ -356,6 +356,7 @@ class Cf3PromisesFileWriterServiceImpl(
          * and "error/technical details"
          */
         val completeErrorMsg = ( s"The generated promises are invalid!errormessage!cf-promise check fails for promises generated at '${paths.newFolder}'"
+                               + "<-Test command is: '" + checkCommand + "'"
                                + (if(errors.isEmpty) "" else errors.mkString("<-", "<-", ""))
```

Comment:
This change should work because the "client side" cuts the message up using the "<-" separator, to replace it by a new line (```<br/>```).